tests/Analytics/AnalyticsTest.php

```diff
@@ -16,9 +16,6 @@ class AnalyticsTest extends TestCase
     /** @var \Utopia\Analytics\Adapter\GoogleAnalytics */
     public $ga;
 
-    /** @var \Utopia\Analytics\Adapter\ActiveCampaign|null */
-    public $ac;
-
```

Comment:
Do we need to remove the adapter itself too?

Reply:
Just asked Christy, we'll keep the Adapter in there for now